### DISCUSSION TITLE
Does Public Bucket really mean Public?

### DISCUSSION THREAD

**Discussion by technologyconsulting** [supabase/supabase#20665] [Questions] [answered]
Hey All,

When we say Public Bucket, do we mean anybody in the world can search, find and browse the contents of the Public Bucket or is it limited to actual app and or the url of the asset if provided (assuming again this can't be discovered by the general public?).

Thanks for the assist in understanding.

**Comment by GaryAustin1** [comment, marked as answer]
Public bucket only applies to reading the file from the public url. No access to list or upload is provided without you adding policies. If your paths are hard to guess (like a uuid) then unless the url is shared the file is unlikely to be seen.

**Reply by technologyconsulting** [reply]
Thanks Gary, I just wanted to be 100% sure.
